packages/fixtures/src/serverless-tester.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { events, ServerlessTester } from '@slack-wrench/fixtures';
+import { App, ExpressReceiver } from '@slack/bolt';
+
+describe('Serverless Tester', () => {
+  let app: App;
+  let receiver: ExpressReceiver;
+  let handler: ServerlessTester;
+  let listener: jest.Mock;
+  let expressMiddleware: jest.Mock;
+
+  const command = '/command';
+
+  const setupFixtures = (endpoints?: Record<string, string>) => {
+    const signingSecret = '';
+    receiver = new ExpressReceiver({ signingSecret, endpoints });
+
+    app = new App({ receiver, token: '' });
+
+    listener = jest.fn(({ ack }) => {
+      ack();
+    });
+
+    expressMiddleware = jest.fn((req, res) => res.send(200));
+
+    app.command(command, listener);
+    handler = new ServerlessTester(receiver.app, signingSecret);
+
+    receiver.app.use('/', expressMiddleware);
+  };
+
+  it('can be used by a bolt app', async () => {
+    expect.assertions(2);
+
+    setupFixtures();
+
+    const result = await handler.sendSlackEvent(events.slashCommand(command));
+
+    expect(listener).toHaveBeenCalled();
+    expect(result.statusCode).toEqual(200);
+  });
```

Comment:
It might be worth a note here that this also tests that the wrapper works with the default route? not too big a deal, but it's implicitly testing more than just what the test says